### PR TITLE
To fix HTMLProofer

### LINKED ISSUE #492: HTMLProofer reveals 3 broken links, after that we can make it fail the build!
Below are the 3 last HTML proofer errors 🎉 
```
- ./topics/assembly/tutorials/debruijn-graph-assembly/tutorial.html
  *  External link https://galaxyproject.github.io/training-material/topics/assembly/slides failed: 404 No error
- ./topics/proteomics/tutorials/secretome-prediction/tutorial.html
  *  linking to internal hash #nut_and_bolt-comment-customizing-the-workflow that does not exist (line 133)
     <a href="#nut_and_bolt-comment-customizing-the-workflow">below</a>
- ./topics/sequence-analysis/index.html
  *  External link http://doi.org/10.5281/zenodo.218574 failed: 410 No error
htmlproofer 3.7.2 | Error:  HTML-Proofer found 3 failures!
```

Once fixed, we can update the `.travis.yml` file as follows:

```diff
diff --git a/.travis.yml b/.travis.yml
index f227a20..df60801 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -41,9 +41,7 @@ script:
   then
     htmlproofer --http-status-ignore 405,999 --url-ignore "/.*localhost.*/","/.*vimeo\.com.*/" --file-ignore "/.*\/files\/.*/"
   else
-    make build
-    # TODO: remove `|| return 0` when htmlproofer passes for the first time
-    make check || return 0
+    make check
   fi
 
 after_success:
```

### PR DESCRIPTION
**Pull request by bebatut**
To fix #492

### CODE REVIEW

File: _layouts/tutorial_hands_on.html [hunk 1]
```diff
@@ -28,9 +28,15 @@
                         </a>
                     </li>
 
-                    {% if tutorial.slides != "" %}
+                    {% if tutorial.slides == "yes" %}
                     <li class="nav-item">
-                        <a class="nav-link" href="{{ site.url }}/topics/{{ topic.name }}/slides" title="Slides for this tutorial">
+                        <a class="nav-link" href="{{ site.url }}/topics/{{ topic.name }}/tutorials/{{ tutorial.name}}/slides" title="Slides for this tutorial">
```
Comment (nsoranzo) [inline _layouts/tutorial_hands_on.html]: Missing space after `tutorial.name` .

Comment (bebatut) [reply _layouts/tutorial_hands_on.html]: Nice catch

**Comment by bebatut** [comment]
I finally removed the commit which is removing the `|| return 0` in Travis. I moved it to another PR

**Comment by willdurand** [comment]
Check the output of travis. It seems arxiv links should be https and there is a weird error related to a js file that has been removed lately. Maybe you should rebase.

**Comment by willdurand** [comment]
Mmh maybe 401 status codes could be ignored or yield warnings only.

**Comment by willdurand** [comment]
Or maybe try this first: https://github.com/gjtorikian/html-proofer/issues/376#issuecomment-284475461

**Comment by willdurand** [comment]
@bebatut err sorry, it looks like this command was already in the `.travis.yml` file 😢

**Comment by bebatut** [comment]
I remove the last commit then...

**Comment by willdurand** [comment]
So this seems to be an error due to Nature.com proxies... I can reproduce it locally:

```
❯ http --follow --headers https://www.nature.com/nrg/journal/v12/n6/full/nrg2986.html
HTTP/1.1 401 Unauthorized
```

**Comment by willdurand** [comment]
Oh wait, but this is a legit error actually. This article cannot be accessed:

> To read this article in full you may need to log in, make a payment or gain access through a site license (see right).

**Comment by willdurand** [comment]
> HTML-Proofer finished successfully.

😎  ship it!

**Pull request merged**